packages/data/README.md

```diff
@@ -381,6 +381,10 @@ _Returns_
 
 Undocumented declaration.
 
+<a name="RegistryContext" href="#RegistryContext">#</a> **RegistryContext**
```

Comment:
Can we make sure this is documented? See also #15176, which proposes documentation for `RegistryConsumer` and `RegistryProvider`.